src/System.Security.Cryptography.Xml/pkg/System.Security.Cryptography.Xml.pkgproj

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <ProjectReference Include="..\ref\System.Security.Cryptography.Xml.builds">
+      <SupportedFramework>netcoreapp1.1;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
>And if we're going to facade on NetFx I think that should be declared here, too...
  
It should. I left the net463 build out of the DirectoryServices port to minimize the amount of steps needed to get the core windows building and included in build.cmd, but eventually we'll want to be able to build targeting net463 as well.